<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inputdatawindow</class>
 <widget class="QDialog" name="inputdatawindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="itemLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="readingMaterialRadioButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Reading Materials</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="quantitySpinBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of items:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="frequencyComboBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Daily</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Weekly</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Monthly</string>
    </property>
   </item>
  </widget>
  <widget class="QRadioButton" name="confectioneryRadioButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Confectionery</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Confectionery-Item Weight:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="weightSpinBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Reading Material-Item Frequency:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name of items:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Stock Type</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
